internal/services/vpcgw/network.go

```diff
@@ -59,6 +58,14 @@ func ResourceNetwork() *schema.Resource {
 				ValidateDiagFunc: verify.IsUUIDorUUIDWithLocality(),
 				Description:      "The ID of the public gateway DHCP config",
 				ConflictsWith:    []string{"static_address", "ipam_config"},
+				DiffSuppressFunc: func(_, oldValue, newValue string, d *schema.ResourceData) bool {
+					if v, ok := d.Get("ipam_config").([]interface{}); ok && len(v) > 0 {
+						return true
+					}
+
+					return oldValue == newValue
+				},
+				Deprecated: "Please use ipam_config",
```

Comment:
Add a link to the migration guide with a more explicit guide that IPAM is the way forward.